categories/urls.py

```diff
@@ -1,7 +1,11 @@
 from django.urls import path
 
+<<<<<<< HEAD
+from .views      import SubCategoriesView         
+=======
 from .views      import SubCategoriesView
 
+>>>>>>> c71405cbf8665fabb85974d61d0a1dd3b353c687

```

Comment:
컨플릭트 해결 안된 부분이 있네요? 화살표 삭제하고 다시 커밋해서 올려주세요~

```suggestion
from .views      import SubCategoriesView         
from .views      import SubCategoriesView

```

Reply:
> 컨플릭트 해결 안된 부분이 있네요? 화살표 삭제하고 다시 커밋해서 올려주세요~

user.py 에 있는 category url 확인했습니다 ㅜ